### PR TITLE
 Auto-generate jobs for openshift/openshift-ansible 

### PR DESCRIPTION
**Pull request by stevekuznetsov**
/assign @smarterclayton

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-master-presubmits.yaml [hunk 1]
```diff
@@ -393,8 +425,6 @@ presubmits:
             configMapKeyRef:
               key: master.yaml
               name: ci-operator-openshift-openshift-ansible
-        - name: HOME
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-master-presubmits.yaml]: Why do you need this for unit tests @vrutkovs ?

Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-master-presubmits.yaml]: One of the test is verifying syntax of ansible playbook, which is ansible call. Ansible wants `$HOME` to store facts

Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-master-presubmits.yaml]: Let's not throw anything away -- why don't you set a temp `$HOME` in your test? Origin tests do that all the time.

**Comment by stevekuznetsov** [comment]
This only breaks out the images presubmit and fixes up some inconsistent args, LGTm

**Pull request merged**